sass/cd/cd-header/_cd-user-menu.scss

```diff
@@ -141,17 +140,6 @@
   }
 }
 
-/* Last block in Global Header should have its dropdown right aligned */
```

Comment:
@rupl I got to this too late, but I'm wondering if removing this CSS rule was intentional?

Reply:
@left23 yes intentional. The entire block of styles was duplicated and the linter told me about the duplicated selector when I was finalizing the PR. I can't comment directly on the line because it's not close enough to the diff but here's the block:

https://github.com/UN-OCHA/common_design/pull/366/files#diff-e3e7a9f9f60276a5348104bf00f644b49dadadc4a3c302195dc1f6135e40098cR132

I just double-checked and it does seem like it was two identical blocks of styles. So I think we're good.